mod_test.go

```diff
@@ -0,0 +1,33 @@
+package testutil
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestWithModules(t *testing.T) {
+	t.Parallel()
+
+	t.Run("The line directive is appended to the go source codes", func(t *testing.T) {
+		testdata := WithModules(t, analysistest.TestData(), nil)
+		tests := []struct {
+			path string
+			want string
+		}{
+			{filepath.Join(testdata, "src", "a", "a.go"), "//line src/a/a.go:1"},
```

Comment:
The difference from #9  is that the path is not absolute (does not have `/` prefix).